javaparser-core/src/main/java/com/github/javaparser/ast/type/UnionType.java

```diff
@@ -53,7 +53,7 @@
  * } catch(<b>IOException | NullPointerException ex</b>) {
  * ...
  * }
- * </pre>
+ * </code></pre>
```

Comment:
Double check this -- the closing `</code>` tag has been removed, but the opening one is still there.

Reply:
This is the addition of a missing `</code>` tag, not the removal of one.  The `<code>` tag is already there.

Reply:
Oops, yes you're right - my bad! 

Would you mind converting it to `{@code }` instead please?

Reply:
Scratch that, let's get it merged in and we can do additional tweaks like that in another PR